esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6427,7 +6427,7 @@ bool CWsDfuEx::onDFUFilePublish(IEspContext &context, IEspDFUFilePublishRequest
         const char *clusterName = fileIdItems.item(1);
         const char *tempFileName = fileIdItems.item(2);
         bool compressed = false;
-        if (fileIdItems.ordinality()>=3)
+        if (fileIdItems.ordinality()>3)
```

Comment:
this is an unrelated bug fix by the looks of it?
i.e. would have hit an exception if fileIdItems only had 3 items, when trying to do fileIdItems.item(3)

Reply:
I see the 'Also fix a bug in ESP DFUFilePublish.' worth expanding on that a bit for the git history